tests/programs/custom/perf_counters_instructions/perf_counters_instructions.c

```diff
@@ -87,8 +87,6 @@ int main(int argc, char *argv[])
                  EVENT_WB_DATA_STALL = 1 << 15 };
 
 
-  __asm__ volatile(".option rvc");
```

Comment:
Did it cause compilation errors because the following instructions cannot be represented with "rvc"?

Reply:
Failed because this option does not exist in the updated toolchains (+ it did nothing in the first place as rvc was not disabled)